packages/frontend/components/item/image-slider/component.tsx

```diff
@@ -8,38 +8,58 @@ interface ItemImageSliderProps {
 
 export const ItemImageSlider = ({ imageLinks }: ItemImageSliderProps) => {
   const [focusedImage, setFocusedImage] = useState(0);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const imagesWrapper = useRef<HTMLDivElement>(null);
 
   const handleClick = (imageLink) => {
     setFocusedImage(imageLink);
   };
 
+  useEffect(() => {
+    function handleWheelScroll(e) {
+      e.preventDefault();
+      imagesWrapper.current.scrollLeft += e.deltaY;
+      imagesWrapper.current.scrollTop += e.deltaY;
+    }
+    imagesWrapper.current.addEventListener('wheel', handleWheelScroll);
```

Comment:
you need to remove an event listener on willUnMount

Reply:
and what does it do?

Reply:
add posibility to scroll horizontal scroll bu wheel

Reply:
doesn't it work with shift/ctrl pressed?